.travis.yml

```diff
@@ -1,11 +1,19 @@
+rvm:
+  - ruby-2.3.5-p0
+
 sudo: required
 services:
   - docker
 
+before_install:
+  - gem install bundler
```

Comment:
Since we had some issues with WA, I advise you to pin the version here with the one you would have in your https://github.com/Liaison-Intl/upgrade_toolbelt/blob/master/Gemfile.lock#L78